content/300-guides/200-deployment/201-serverless/300-deploy-to-vercel.mdx

```diff
@@ -300,21 +66,10 @@ In this section you associate a second PostgreSQL database with a Vercel preview
 
 5. Click **Save**.
 
-### Result
-
-You should see the `DATABASE_URL` environment variable for the preview environment in your list of environment variables:
-
-![List of environment variables](./images/300-70-deploy-to-vercel-environment-variables.png)
-
-## Summary
-
-You have successfully deployed your application to Vercel.
-
-For more insight into the Prisma Client API, look at the function handlers in the [`api/` directory](https://github.com/prisma/deployment-example-vercel/tree/main/pages/api) of the example repository.
+## Connection pooling
 
-<Admonition type="info">
+When you use a Function-as-a-Service provider, like Netlify, it is beneficial to pool database connections for performance reasons. This is because every function invocation may result in a new connection to your database which can quickly run out of open connections.
```

Comment:
```suggestion
When you use a Function-as-a-Service provider, like Vercel Serverless functions, pooling database connections for performance reasons is beneficial. This is because every function invocation may result in a new connection to your database which can quickly run out of open connections.
```